python/knowhere/knowhere.i

```diff
@@ -354,4 +356,51 @@ DumpRangeResultDis(knowhere::DataSetPtr result, float* dis, int len) {
     }
 }
 
+void
+Dump(knowhere::BinarySetPtr binset, const std::string& file_name) {
+    auto binary_set = *binset;
+    auto binary_map = binset -> binary_map_;
+    std::ofstream outfile;
+    outfile.open(file_name, std::ios::out | std::ios::trunc);
+    for (auto it = binary_map.begin(); it != binary_map.end(); ++it) {
```

Comment:
please check is the file was opened successfully via `if (outfile.good()) {}`

Reply:
fixed